app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,4 @@
 <resources>
     <string name="app_name">Blooddonor-android</string>
+    <string name="description" />
```

Comment:
What's the purpose of this line?

Reply:
This is unnecessary code. To be removed.